### PR TITLE
add corosync

### PR DESCRIPTION
**Pull request by cpg1111**

### CODE REVIEW

File: roles/maas_corosync/template/corosync.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,41 @@
+system {
```
Comment (ElineMaaikedeWeerd) [inline roles/maas_corosync/template/corosync.conf.j2:1]: folder name template > templates?

File: roles/maas_corosync/template/corosync.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    debug: off
+    logger_subsys {
+        subsys: QUORUM
+        debug: off
+    }
+}
+
+
+nodelist {
+{% for i, pacemaker_host in enumerate(groups['pacemaker']) %}
```
Comment (ElineMaaikedeWeerd) [inline roles/maas_corosync/template/corosync.conf.j2:27]: I'm not sure if enumerate exists as a filter, but ignore if I'm wrong about that

Comment (ElineMaaikedeWeerd) [inline roles/maas_corosync/template/corosync.conf.j2:27]: I'm not sure if enumerate exists as a filter, but ignore if I'm wrong about that

File: site.yaml [hunk 3]
```diff
@@ -19,6 +19,15 @@
       maas_use_version: "{{ maas_tmp_ver.stdout | regex_search('[0-9]+.[0-9]+') if maas_version is not regex('[0-9]+.[0-9]+(.[0-9]+)?') else maas_version }}"
       cacheable: true
 
+- hosts: maas_corosync:
```
[the remark this answers is not in the document]
Comment (ElineMaaikedeWeerd) [reply site.yaml:22]: fixing the trailing colon will also make the linter happy :-)

File: roles/maas_corosync/template/corosync.conf.j2 (hunk 2 above)
Comment (cpg1111) [reply roles/maas_corosync/template/corosync.conf.j2:27]: ah you are correct, will update.

**Review by ElineMaaikedeWeerd — APPROVED** [review]
looks good to me!

**Pull request merged**